VirtoCommerce.CustomerModule.Data/Handlers/LogChangesMemberChangedEventHandler.cs

```diff
@@ -0,0 +1,204 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using AutoCompare;
+using VirtoCommerce.Domain.Commerce.Model;
+using VirtoCommerce.Domain.Common.Events;
+using VirtoCommerce.Domain.Customer.Events;
+using VirtoCommerce.Domain.Customer.Model;
+using VirtoCommerce.Platform.Core.ChangeLog;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Core.Events;
+
+namespace VirtoCommerce.CustomerModule.Data.Handlers
+{
+    public class LogChangesMemberChangedEventHandler : IEventHandler<MemberChangedEvent>
+    {
+        private readonly IChangeLogService _changeLogService;
+        private readonly string[] _observedProperties;
+
+        public static string[] ObservedProperties()
```

Comment:
This method name should starts with Get prefix. `GetDefaultObservedProperties`

Reply:
I don't agree, but if you insist, I'll change it.

Reply:
I'll change it to `DefaultObserverProperties` static property